tutorials/polkadot-sdk/system-chains/asset-hub/asset-conversion.md

```diff
@@ -9,7 +9,7 @@ description: A guide detailing the step-by-step process of converting assets on
 
 Asset Conversion is an Automated Market Maker (AMM) utilizing [Uniswap V2](https://github.com/Uniswap/v2-core){target=\_blank} logic and implemented as a pallet on Polkadot's Asset Hub. For more details about this feature, please visit the [Asset Conversion on Asset Hub](https://wiki.polkadot.network/docs/learn-asset-conversion-assethub){target=\_blank} wiki page.
 
-This guide will provide detailed information about the key functionalities offered by the [Asset Conversion](https://github.com/paritytech/polkadot-sdk/tree/master/substrate/frame/asset-conversion){target=\_blank} pallet on Asset Hub, including:
+This guide will provide detailed information about the key functionalities offered by the [Asset Conversion](https://github.com/paritytech/polkadot-sdk/tree/polkadot-stable2412/substrate/frame/asset-conversion){target=\_blank} pallet on Asset Hub, including:
```

Comment:
You can use variables in link text, so all these links should be updated. For example this one should be:

```suggestion
This guide will provide detailed information about the key functionalities offered by the [Asset Conversion](https://github.com/paritytech/polkadot-sdk/tree/{{dependencies.polkadot_sdk.stable_version}}/substrate/frame/asset-conversion){target=\_blank} pallet on Asset Hub, including:
```